test/test_axes.jl

```diff
@@ -88,3 +88,30 @@ end
     p = plot(1:2, xtickor = :out)
     @test p[1][:xaxis][:tick_direction] === :out
 end
+
+@testset "aliases" begin
+    p = plot(1:2, guide = "both labels")
+    @test p[1][:xaxis][:guide] === p[1][:yaxis][:guide] === "both labels"
+    p = plot(1:2, label = "test")
+    @test p[1][:xaxis][:guide] === p[1][:yaxis][:guide] === ""
+    p = plot(1:2, lim = (0, 3))
+    @test xlims(p) === ylims(p) === (0,3)
+    p = plot(1:2, tick = [1.25, 1.5, 1.75])
+    @test p[1][:xaxis][:ticks] == p[1][:yaxis][:ticks] == [1.25, 1.5, 1.75]
+    p = plot(1:2, labelfontsize = 4)
+    @test p[1][:xaxis][:guidefontsize] == p[1][:yaxis][:guidefontsize] == 4
+    p = plot(1:2, gα = .07)
+    @test p[1][:xaxis][:gridalpha] ≈ p[1][:yaxis][:gridalpha] ≈ .07
+    p = plot(1:2, gridls = :dashdot)
+    @test p[1][:xaxis][:gridstyle] === p[1][:yaxis][:gridstyle] === :dashdot
+    p = plot(1:2, gridcolor = :red)
+    @test p[1][:xaxis][:foreground_color_grid] === p[1][:yaxis][:foreground_color_grid] === RGBA{Float64}(1.,0.,0.,1.)
+    p = plot(1:2, minorgridcolor = :red)
+    @test p[1][:xaxis][:foreground_color_minor_grid] === p[1][:yaxis][:foreground_color_minor_grid] === RGBA{Float64}(1.,0.,0.,1.)
+    p = plot(1:2, grid_lw = .01)
+    @test p[1][:xaxis][:gridlinewidth] ≈ p[1][:yaxis][:gridlinewidth] ≈ .01
+    p = plot(1:2, minorgrid_lw = .01)
+    @test p[1][:xaxis][:minorgridlinewidth] ≈ p[1][:yaxis][:minorgridlinewidth] ≈ .01
+    p = plot(1:2, tickor = :out)
+    @test p[1][:xaxis][:tick_direction] === p[1][:yaxis][:tick_direction] === :out
```

Comment:
Would be good to also have tests for z-Axis

Reply:
True that.

Is it okay to introduce functions like
```julia
xyzeq(p::Plot, s::Symbol, val) = p[1][:xaxis][s] == p[1][:yaxis][s] == p[1][:zaxis][s] == val
xyzid(p::Plot, s::Symbol, val) = p[1][:xaxis][s] === p[1][:yaxis][s] === p[1][:zaxis][s] === val
xyzsim(p::Plot, s::Symbol, val) = p[1][:xaxis][s] ≈ p[1][:yaxis][s] ≈ p[1][:zaxis][s] ≈ val
```
inside the testset for this purpose or should such a thing be avoided and be added explicitly?

Reply:
Looks more readable so ok for me.
Note that I've used `≈` for floats (good practice), but it's not mandatory in that case since there are no ops on floats.